src/xcode/ENA/ENA/Source/Services/TicketValidation/MockTicketValidation.swift

```diff
@@ -17,7 +17,7 @@ final class MockTicketValidation: TicketValidating {
 	}
 
 	let initializationData: TicketValidationInitializationData
-
+	var allowList = TicketValidationAllowList(validationServiceAllowList: [], serviceProviderAllowList: [])
```

Comment:
```suggestion
	var allowList = TicketValidationAllowList(validationServiceAllowList: [], serviceProviderAllowList: [])
	
```

Would be nice to have a newline before the func :)